src/main/java/com/happystays/cqrs/core/dto/events/PnrEventModel.java

```diff
@@ -1,4 +1,4 @@
-package com.happystays.cqrs.core.dto.events.eventmodel;
+package com.happystays.cqrs.core.dto.events;
 
 import lombok.Builder;
 import lombok.Data;
```

Comment:
Please keep PnrEventModel and EventModel inside com.happystays.cqrs.core.events -> eventstoreModel -> PnrEventModel.Java